test-with-node-testrunner/test/01-spies.test.ts

```diff
@@ -0,0 +1,33 @@
+import {
+    describe,
+    it,
+    mock
+} from 'node:test';
+import assert from 'node:assert';
+
+
+function run({fn, times}){
```

Comment:
Formatting seems off here, check out [prettier](https://prettier.io/)  including instructions to setup  [VSCode](https://prettier.io/docs/editors#visual-studio-code)